charts/alfresco-adf-app/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.configMap }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "alfresco-adf-app.fullname" . }}
```

Comment:
I am thinking that maybe we can hardcode the name of that configmap to make it easier for users to mount it. Not sure

Reply:
If user wants to mount his own cm I think we agreed he would:
* create his own cm
* create an extra volume for it (where he can use whatever name he wants)
* create the volume mount that goes with it
and so we would not have this configMap at all

If we agree on that, then we should document it in this chart